lib/static/new-ui/features/suites/components/TestStatusFilter/selectors.ts

```diff
@@ -0,0 +1,52 @@
+import {createSelector} from 'reselect';
+import {getAllBrowserIds, getResults} from '@/static/new-ui/store/selectors';
+
+export interface StatusCounts {
+    success: number;
+    fail: number;
+    skipped: number;
+    total: number;
+    retried: number;
+    retries: number;
+    idle: number;
+}
+
+export const getStatusCounts = createSelector(
+    [getResults, getAllBrowserIds],
+    (results, browserIds) => {
+        const latestAttempts: Record<string, {attempt: number; status: string, timestamp: number}> = {};
+        const retriedTests = new Set<string>();
+
+        let retries = 0;
+        Object.values(results).forEach(result => {
+            const {parentId: testId, attempt, status, timestamp} = result;
+            if (attempt > 0) {
```

Comment:
can we check it by `status === RETRIED`?

Reply:
No, status stores an actual test status like fail, error or success. There's no such thing as "retried" status in sqlite db or anywhere else.